en/clubs/prylmangleriet/body.md

```diff
@@ -1,69 +1,55 @@
-# Prylmånglaren
+# Prylmånglariet
 
-Tjo bästis!
+What is Prylmångeriet actually?
 
-Välkommen till Prylmånglarens sida på Datasektionen. Om du inte redan vet vem jag är (du är fortfarande min bästis) heter jag Lisa Lund och är Prylis under 2015. Jag tar hand om sektionens märken, ovvar och andra otroligt balla grejer. Målet är att nästan exakt hela tiden anordna försäljningar, framför allt i __META__, men jag finns även på vissa evenemang och säljer relevanta prylar (så kallad Prylis-to-go). Försäljningar meddelas via hemsidan, facebook, och i desperata fall, via megafon i sektionslokalen. Om ni inte hittar mig och har ett sjukt akut behov av mina tjänster, skicka ett [mail](mailto:prylis@d.kth.se) eller ring eller smsa eller nåt på __076-213 33 96__.
+Well... Prylmångleriet is the club at the Computer Science chapter, lead by Prylmånglaren (also called Prylis). Prylmångleriet handles all from embroidered patches, boilersuits (Ovve) and all other type of merch for the chapter! Our work mainly consists of designing new patches and the sale of them but also design of the chapter's songbook. Contact Prylis by [mail](mailto:prylis@d.kth.se) if you have any project ideas that you want to realize or if you have any questions.
```

Comment:
```suggestion
Well... Prylmångleriet is a club at the Computer Science chapter, lead by Prylmånglaren (also called Prylis). Prylmångleriet handles all from embroidered patches, boilersuits (Ovve) and all other type of merch for the chapter! Our work mainly consists of designing new patches and the sale of them but also designing the chapter's songbook. Contact Prylis by mail at [prylis@d.kth.se](mailto:prylis@d.kth.se) if you have any project ideas that you want to realize or if you have any questions.
```
Some grammar.
I think it is better to write out the whole mail to make things simpler if mailing clients are not correctly set up.

Reply:
Yup sounds good! My grammar wasn't tip-top so thanks for checking that and yeah might be good to write it out.